### PR TITLE
Doc 8288 v2

### PR DESCRIPTION
**Pull request by dlutzCoveo**
New branch to merge with 0.1.0

https://codesandbox.io/s/vigilant-tdd-pich4

### CODE REVIEW

File: src/Components/CategoryFacet.tsx
Comment (kaitlynkatz) [inline src/Components/CategoryFacet.tsx:81]: This is _super_ nitpicky, just because the whole point of this example is that people will look through the code, so our comments aren't just for us
```suggestion

  // Calculates the left position offset for values depending on if there are any parent values.
```

File: src/Components/CategoryFacet.tsx
Comment (kaitlynkatz) [inline src/Components/CategoryFacet.tsx:82]: In fact, the name of this method could probably be changed, because it's not the parents we're aligning, it's the values. Maybe `valuesAlign` ?

File: src/Components/CategoryFacet.tsx
Comment (kaitlynkatz) [inline src/Components/CategoryFacet.tsx:127]: I don't think this comment is necessary as long as the method it calls is clearly commented

**Review by kaitlynkatz — APPROVED** [review]
Very nice! I worked on it with you though, so definitely wait for another review 😄

File: src/Components/CategoryFacet.tsx
Comment (fbeaudoincoveo) [inline src/Components/CategoryFacet.tsx:10]: You can remove this

File: src/Components/CategoryFacet.tsx
Comment (fbeaudoincoveo) [inline src/Components/CategoryFacet.tsx:51]: Maybe we could add a comment here to give an example of what a typical `atlgeographicalhierarchy` field value looks like

For instance, the Washington, D.C. result would have `North America;North America|United States;` as an `atlgeographicalhierarchy` field value

File: src/Components/CategoryFacet.tsx
Comment (fbeaudoincoveo) [inline src/Components/CategoryFacet.tsx:169]: I noticed that there's a warning in the console related to the Category Facet:

Screen Shot 2021-02-15 at 12 20 13

@kaitlynkatz , @jpmarceau rings a bell?

**Review by fbeaudoincoveo — APPROVED** [review]

File: src/Components/CategoryFacet.tsx
Comment (dlutzCoveo) [reply src/Components/CategoryFacet.tsx:51]: I've added:
// a typical atlgeographicalhierarchy field value for a city would appear: Continent;Continent|Country;
    // for example Washington, DC. would be: North America;North America|United States;

File: src/Components/CategoryFacet.tsx
Comment (jpmarceau) [reply src/Components/CategoryFacet.tsx:168]: Oh yeah, this happens all the time 🙃 
We just need to provide keys on list items or React is very very sad. In this case, we have to add it at like 108, I'll comment there 👍 

(I'm kinda mad at myself for not answering this question earlier btw, considering how simple it is 😅 I saw the comment in my emails but completely forgot 😭)

File: src/Components/CategoryFacet.tsx
Comment (jpmarceau) [inline src/Components/CategoryFacet.tsx:108]:
```suggestion
          key={parent.value}
          onClick={() =>
```

Comment (jpmarceau) [reply src/Components/CategoryFacet.tsx:108]: This will fix the console error :)

**Review by jpmarceau — APPROVED** [review]
GG!

**Pull request merged**
